tomcat-jdbc/pom.xml

```diff
@@ -0,0 +1,150 @@
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.camel.springboot.example</groupId>
+        <artifactId>examples</artifactId>
+        <version>4.4.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>camel-example-spring-boot</artifactId>
+    <name>Camel SB Examples :: Tomcat JDBC</name>
+    <description>An example showing how to deploy a Camel Spring Boot application in Tomcat using its JDBC Data Source</description>
+    <packaging>war</packaging>
+
+    <properties>
+        <category>Beginner</category>
+
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <!-- Camel BOM -->
+            <dependency>
+                <groupId>org.apache.camel.springboot</groupId>
+                <artifactId>camel-spring-boot-bom</artifactId>
+                <version>${camel-version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <!-- Spring Boot BOM -->
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>${spring-boot-version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+
+        <!-- Spring Boot -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-actuator</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-jdbc</artifactId>
+            <exclusions>
+                <exclusion>
+                    <!-- Spring Boot uses HikariCP DataSource by default, so we remove it -->
+                    <groupId>com.zaxxer</groupId>
+                    <artifactId>HikariCP</artifactId>
```

Comment:
Does it mean that no pooling will be used by removing Hikari? or Spring Boot will fallback to other pooling options?

If you want to introduce pooling, you can use agroal that is supported by Red Hat and suggested in other examples like https://github.com/jboss-fuse/camel-spring-boot-examples/blob/camel-spring-boot-examples-4.4.0-branch/spring-boot-jta-jpa-autoconfigure/pom.xml#L134

Reply:
It's using Tomcat's pooled datasource.

https://github.com/jboss-fuse/camel-spring-boot-examples/pull/145/files#diff-004fd9875f58addca5193e1af6df9a0421788fbbfb8768d1799fea8c19915d67R21

https://github.com/jboss-fuse/camel-spring-boot-examples/pull/145/files#diff-004fd9875f58addca5193e1af6df9a0421788fbbfb8768d1799fea8c19915d67R21

Reply:
oh! okay, great then